### PR TITLE
[5.1] Upgraded To PsySH 0.5

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Replaces #9336.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Ping @nicolas-grekas. This doesn't seem to actually work.

image

**Comment by GrahamCampbell** [comment]
Ok, got that fixed. All down to typos. :)

**Comment by taylorotwell** [comment]
Put those Casters in a Tinker namespace or something.

**Comment by taylorotwell** [comment]
Illuminate\Console\Tinker

**Comment by GrahamCampbell** [comment]
There is only one caster class though, so I didn't think it made sense?

File: src/Illuminate/Foundation/Console/IlluminateCaster.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+     * @param  \Illuminate\Foundation\Application  $app
+     * @param  array  $a
+     * @param  Stub  $stub
+     * @param  bool  $isNested
+     * @param  int  $filter
+     * @return array
+     */
+    public static function castApplication(Application $app, array $a, Stub $stub, $isNested, $filter = 0)
+    {
+        $a = [];
```
Comment (taylorotwell) [inline src/Illuminate/Foundation/Console/IlluminateCaster.php]: Use a more readable variable name here please :smile:

**Pull request merged**